scripts/increment_auto_off.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
+"""A program that ticks a counter down until deactivating log_persistence"""
+
+import logging
+import sys
+import json
+import re
+
+import configparser
+import requests
+import subprocess
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+sh = logging.StreamHandler(sys.stdout)
+logger.addHandler(sh)
+
+state_persist: bool
+state_delay: int
+
+api_read_failure = False
+try:
+  response = requests.get('http://localhost:5001/settings/persist_logs', timeout=10)
+  state = response.json()
+
+  state_persist = state["persist_logs"]
+  state_delay = state["auto_off_delay"]
+except KeyError as exc:
+  logger.exception("Unable to read log persistence state through the api, attempting to read directly...")
+  api_read_failure = True
+except Exception as exc:
+  logger.exception(f"increment_auto_off.py has failed with the following exception:\n{exc}")
+  api_read_failure = True
+
+
+if api_read_failure:
+  try:
+    # If the api is inaccessible, assume it is down for whatever reason and extrapolate that there is no race condition to reading it directly
+    # This section is a copy of the /settings/persist_logs GET in asgi.py
```

Comment:
if this is a copy of any section of the code, you should probably create a function out of it and call it from both places

Reply:
This is difficult due to them being two entirely separate programs unfortunately

I tried doing something like adding a symbolic link between the amplipi directory and `/usr/local/bin` where this script ends up so I could do a relative import and that almost worked except for the fact that this script is outside of the venv and so doesn't have the dependencies to import anything without errors

Maybe someday I'll come back to this and make it even better, for now it's been two hours since you approved this PR so I'm gonna stop spending time on making it more than it is